patroni/ctl.py

```diff
@@ -921,8 +933,8 @@ def remove(cluster_name: str, group: Optional[int], fmt: str) -> None:
     dcs = get_dcs(cluster_name, group)
     cluster = dcs.get_cluster()
 
-    if is_citus_cluster() and group is None:
-        raise PatroniCtlException('For Citus clusters the --group must me specified')
+    if is_mpp_cluster() and group is None:
+        raise PatroniCtlException('For {0} clusters the --group must me specified'.format(_get_mpp().type))
```

Comment:
let's better use f-strings for the new code
```suggestion
        raise PatroniCtlException(f'For {_get_mpp().type} clusters the --group must me specified')
```